src/iconv.jl

```diff
@@ -2,7 +2,7 @@
 
 module iconv
 import Base: close, eof, flush, read, readall, write
-import Base.Libc: errno, strerror, E2BIG, EINVAL, EILSEQ
+import Base.Libc: errno, strerror, E2BIG, EINVAL, EILSEQ, EOVERFLOW
```

Comment:
Are you sure this is needed? EOVERFLOW = 84 on Mac, and EILSEQ = 84 on Linux, which is why I didn't catch the correct error in the previous code (where I defined the constants by hand). Now that I use constants from Libc, this should work fine (as can be seen from the fact that the tests pass).